README.md

```diff
@@ -16,7 +16,12 @@ In the future we might want to split up the project,  similar as it has been don
 - Add `VoodooPS2Controller.kext`
 - Add the patch in `config.plist.patch`, so `VoodooPS2Controller` does not attach itself to the PS2 interface of the touchpad
 - Delete `VoodooPS2Controller.kext/Contents/PlugIns/VoodooPS2Mouse.kext`, otherwise the touchpad will not work after sleep
-- Add the four patches found here https://github.com/leo-labs/macOS-ThinkPad-T480s/blob/master/EFI/CLOVER/config.example.plist#L436-L467 and https://github.com/leo-labs/macOS-ThinkPad-T480s/blob/master/EFI/CLOVER/config.example.plist#L500-L535 to supress loading of those kexts
+- Add the four patches found here to supress loading of those kexts:
+  - https://github.com/leo-labs/macOS-ThinkPad-T480s/blob/master/EFI/CLOVER/config.example.plist#L436-L467
+  - https://github.com/leo-labs/macOS-ThinkPad-T480s/blob/master/EFI/CLOVER/config.example.plist#L500-L535 
+- In some laptops it's necessary to install the DSDT to make the kext working.
```

Comment:
I'd avoid telling people to add their DSDT. It causes issues with OpenCore, and is just an eh solution in general. Might be better to just tell people to check their DSDT for a _STA method under their SBUS device. If there is some sort of `STAS` variable or `OSYS` check, maybe try XOSI. This page works well: https://dortania.github.io/Getting-Started-With-ACPI/Laptops/trackpad.html. Though, you'd be looking for an SBUS device rather than GPIO.

Reply:
I think Hackintosh Slav was gonna do some cleanup, so it might be better to wait a bit on this addition.

Reply:
I'm going to see further about this, I'm still in clover but I can start testing on opencore too

Reply:
![Screen Shot 2020-05-29 at 3 25 01 AM](https://user-images.githubusercontent.com/25161460/83210894-24854b80-a15c-11ea-82f2-a057d4ce4b4a.png)
This is the patch anyway

Reply:
Check [SSDT-OC-XOSI.dsl](https://github.com/daliansky/OC-little/blob/master/04-操作系统补丁/SSDT-OC-XOSI.dsl)
This is the XOSI hot patch, it has the same function compared to this DSDT patch.

Reply:
btw, the link about DSDT patch is wrong.
It should be https://github.com/VoodooI2C/VoodooI2C-Patches/blob/master/Windows/Windows_10.txt

Reply:
XOSI can cause issues with booting other OSes - the above implementation from oc-little seems to take care of that though. XOSI can also interfere with OSID as well on Dell laptops, which can break backlight keys. This can be gotten around with another ACPI patch, but it's a lot of patches at that point. So *generally* I find a more narrow patch to STAS/whatever value that _STA checks is better.

i.e. if you had
```
Method(_STA, 0, NotSerialized) {
  if (STAS == One) {
    Return (0x0f)
  }
  Return (0x0)
}
```
You could make an SSDT similar to
```
Definition Block (/* I forget all this */) {
  If (_OSI("Darwin")) {
    STAS = One
  }
}
```

Maybe we could offer both solutions using XOSI.